src/client/src/rt-components/tear-off/ExternalWindow.tsx

```diff
@@ -43,8 +42,7 @@ const ExternalWindow: FC<ExternalWindowProps> = ({
     const getWindow = async () => {
       externalWindow = await platform.window.open(config, release)
       if (externalWindow) {
-        externalNativeWindow = await externalWindow.getNativeWindow()
```

Comment:
this is what I fixed in recent refactor - I think this should have been a listener on the external window, but it was a listener on a global window. So I fixed it to listen to external. Unfortunately, this lead to introducing a new API - `getNativeWindow`. This API was not implemented in glue42 because `any` was used in the typing (see change in `glue.ts`). THus we were not able to see a compilation error which would have showed us that `getNativeWindow` was not used there. Lesson learned - try to get rid of all `any` - they are evil :)

Reply:
to sum up - I removed the new `getNativeWindow` api in favor of old approach - listening to global window (whatever reason for that was)